packages/scan/tsconfig.lib.json

```
@@ -0,0 +1,11 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "outDir": "../../dist/out-tsc",
+    "declaration": true,
+    "types": ["node"],
+    "lib": ["dom"]
```

Is it intended? What is the purpose?
```suggestion
    "lib": ["dom"]
```